docs/ingestion/data-formats.md

```diff
@@ -128,6 +128,8 @@ The CSV `inputFormat` has the following components:
 
 ### TSV (Delimited)
 
+The `inputFormat` to load data of a delimited format. An example is:
```

Comment:
An example for `inputForm` using JSON data is:

Reply:
Also this phrase "The `inputFormat` to load data of a delimited format. An example is:" keeps being repeated after each type is introduced...it feel repetitive and unecessary

Reply:
Ok. This was fun 😬 Cleaned it up for the most part, PTAL